tls/s2n_x509_validator.c

```diff
@@ -85,47 +87,25 @@ int s2n_x509_trust_store_add_pem(struct s2n_x509_trust_store *store, const char
         store->trust_store = X509_STORE_new();
     }
 
-    s2n_error err = S2N_ERR_DECODE_CERTIFICATE;
-
-    struct s2n_stuffer pem_in_stuffer = {{0}};
-    struct s2n_stuffer der_out_stuffer = {{0}};
-    struct s2n_blob next_cert = {0};
+    _cleanup_(s2n_stuffer_free) struct s2n_stuffer pem_in_stuffer = {{0}};
```

Comment:
I'd like to wrap this in a simple Macro, how about? 

`DEFER_CLEANUP(struct s2n_stuffer pem_in_stuffer = {{0}}, s2n_stuffer_free)`

Reply:
Cando.